orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/purple/PurpleQC.java

```diff
@@ -24,5 +24,7 @@ public interface PurpleQC
 
     int unsupportedCopyNumberSegments();
 
+    int copyNumberSegments();
```

Comment:
Minor but maybe `totalCopyNumberSegments` may be a better name, to distinguish from `unsupportedCopyNumberSegments`? Also, maybe more logical to place this variable above unsupported?